dimod/reference/samplers/exact_solver.py

```diff
@@ -30,7 +30,7 @@
 
 
 class ExactSolver(Sampler):
-    """A simple exact solver for testing and debugging.
+    """A simple exact solver for testing and debugging code using your local CPU.
```

Comment:
I don't think we can emphasize enough this is only for testing and debugging on small-scale problems.

Also, I think it's worth mentioning here `ExactSolver` is a python implementation of exhaustive (brute force) search over all possible samples. (Remember how one of the "influencers" misinterpreted the list of samples returned by exact solver.)